### PR TITLE
Publishes Calico Enterprise 3.20 EP1 

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 1]
```diff
@@ -16,47 +16,33 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
-- [DNS policy for Windows](../getting-started/install-on-clusters/windows-calico/limitations#dns-policy-limitations)
+## Bug fixes
 
-   Use domain names in policies to identify services outside the cluster, which is often operationally simpler and more robust than using IP addresses.
+* WAF events now include source information such as IP and namespace from where the event originated from.
+* Fix that Felix would panic when trying to resync a temporary IP set. Temporary IP sets are created in certain scenarios after previous failures.
 
-<!-- ## Bug fixes -->
 
 <!-- Follow this template: Problem-Cause-Fix-Result -->
```
Comment (sabags) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx:52]: Does this (L47- L59) need to be removed ?

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 2]
```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
-* The anomaly detection feature was removed in v3.18.1 If anomaly detection is enabled and you upgrade to {{prodname}} 3.18, you will stop receiving anomaly detection alerts.
-* [Manual install for Windows](../getting-started/install-on-clusters/windows-calico/manual-install/) will be deprecated in a future release. Starting in v3.18.1, the [standard installation is operator-based](../getting-started/install-on-clusters/windows-calico/operator).
-
-## Technology Preview features
-
-- [Web application firewall](../threat/web-application-firewall)
+:::note Update note
+Updating to Calico Enterprise 3.20 introduces breaking changes to the packet capture and compliance reporting features.
+To continue using these features, you must re-enable them during or after the update.
+* **Multi-cluster management** You must enable compliance reports first in the management cluster, followed by the managed clusters.
```
Comment (vara2504) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx:35]: @ctauchen it is good to have this line above before the Update note. it is applicable for new installation and upgrade scenario both. **Multi-cluster management** You must enable compliance reports first in the management cluster, followed by the managed clusters.

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 3]
```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
-### Deprecated and removed features
-
-* The anomaly detection feature was removed in v3.18.1 If anomaly detection is enabled and you upgrade to {{prodname}} 3.18, you will stop receiving anomaly detection alerts.
-* [Manual install for Windows](../getting-started/install-on-clusters/windows-calico/manual-install/) will be deprecated in a future release. Starting in v3.18.1, the [standard installation is operator-based](../getting-started/install-on-clusters/windows-calico/operator).
-
-## Technology Preview features
-
-- [Web application firewall](../threat/web-application-firewall)
+:::note Update note
+Updating to Calico Enterprise 3.20 introduces breaking changes to the packet capture and compliance reporting features.
```
Comment (vara2504) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx]: I wont say breaking change here.  Also compliance will have no impact.only for packet capture user need to create packetCaptureAPI CR.

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 4]
```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
-- Jira and Slack webhook integration for security event alerts
+For more information, see [Installing Calico Enterprise on an OpenShift HCP cluster](../getting-started/install-on-clusters/openshift/hostedcontrolplanes.mdx)
 
-   By [configuring webhooks](../threat/configuring-webhooks.mdx), you can now push alerts from the Security Overview dashboard in Manager UI to Jira and Slack so incident response and security teams can use native tools to respond to security event alerts.
+### Packet capture and compliance reports are disabled by default
 
-- Added threat feed alerts
+We've changed the default behavior for packet capturing and compliance reporting.
+These features are now disabled by default.
+You can enable either feature in Manager UI or by running a kubectl command.
```
Comment (vara2504) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx]: Nit: It is good to say, they can enable during installation too -
`You can enable either feature during installation, later via the Manager UI, or by running a kubectl command.` but it is ok even if we dont state them explicilty. I will leave that up to you.

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 5]
```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
+For Helm installations, you can enable the features during and update or installation.
```
Comment (vara2504) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx]: This statement sounds to be missing some thing.
For [Helm installations](../getting-started/install-on-clusters/kubernetes/helm#install-calico-enterprise), you can enable the feature during installation or update via `values.yaml` file.

File: calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx [hunk 6]
```diff
@@ -16,47 +18,38 @@ Learn about the new features, bug fixes, and other updates in this release of {{
...
-
-## Technology Preview features
-
-- [Web application firewall](../threat/web-application-firewall)
+:::note Update note
+Updating to Calico Enterprise 3.20 introduces breaking changes to the packet capture and compliance reporting features.
+To continue using these features, you must re-enable them during or after the update.
+* **Multi-cluster management** You must enable compliance reports first in the management cluster, followed by the managed clusters.
+* **PacketCapture CR** To maintain packet capture during an upgrade, you can create a PacketCaptureAPI custom resource.
+* **Helm** Include the `packetcaptureAPI` and `compliance` flags in your `values.yaml` file to keep these features enabled during an update.
```
Comment (vara2504) [inline calico-enterprise_versioned_docs/version-3.20-1/release-notes/index.mdx:37]: The above 2 statement is good enough for upgrade notes. we can move the Multi-cluster management statement generic above.

**Pull request merged**